README.md

```diff
@@ -15,27 +15,43 @@
  - limitations under the License.
  -->
 
-# Apache Kyuubi
-
-<img src="https://svn.apache.org/repos/asf/comdev/project-logos/originals/kyuubi-1.svg" alt="Kyuubi logo" height="120px" align="right" />
+<p align="center">
+  <img src="https://svn.apache.org/repos/asf/comdev/project-logos/originals/kyuubi-1.svg" alt="Kyuubi logo" height="120px"/>
+</p>
+
+<p align="center">
+  <a href="https://kyuubi.apache.org/releases.html">
+    <img src="https://img.shields.io/github/v/release/apache/kyuubi" />
+  </a>
+  <a href="https://github.com/apache/kyuubi/pulls">
+    <img src="https://img.shields.io/github/issues-pr-closed/apache/kyuubi" />
+  </a>
+  <a href="https://github.com/apache/kyuubi/pulse">
+    <img src="https://img.shields.io/tokei/lines/github/apache/kyuubi" />
+  </a>
+  <a href="https://github.com/apache/kyuubi/graphs/contributors">
```

Comment:
The stats of PRs, contributors, and stars seem to have been provided by GitHub on the page

Reply:
Yes, it more about highlights and interations.
The whole line with all the badges comes to "Hi, I'm Kyuubi. I are ready as in latest released version in 1.6.1-incubating and later, which conducted in 2.5k PRs and resulted in 22K total links, and all the commits are done by the community of 116 contributures. Welcome to mark this project starred and see me later."

Reply:
Or any better suggestion for the item and order of badges list?

Reply:
Sound fine to me.

BTW, can we move those badges for project maintenance to https://github.com/apache/kyuubi/blob/master/CONTRIBUTING.md, PMCs somehow rely on them to evaluate the community healthiness

Reply:
Agree, or let's do it in the follow-up PR to bring missing badges to `CONTRIBUTING.md`.